openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -63,7 +67,12 @@ def to_representation(self, user):
         :param user: User object
         :return: Dict serialized account
```

Comment:
I don't think this need to be updated this method still returns Dict of serialized account